enaml/applib/live_editor_model.py

```diff
@@ -232,6 +238,38 @@ def refresh_view(self):
         else:
             self.traceback = ''
 
+    def autocomplete(self, source, position):
+        """ Obtain autocompletion suggestions for the source text 
+        using jedi if available.
+        
+        """
+        if not COMPLETION_AVAILABLE or not source:
+            return []
+        try:
+            #: Use jedi to get suggestions
+            line, column = position
+            script = jedi.Script(source, line+1, column)
```

Comment:
By curiosity, as I have not yet tested this, do this does not slow down too much the edition ?

Reply:
I haven't tested it with any very large files but I haven't noticed any slowdowns. .